src/sentry/api/endpoints/organization_auth_provider_send_reminders.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+
+from django.utils.translation import ugettext_lazy as _
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationAdminPermission
+from sentry.tasks.auth import email_missing_links
+
+ERR_NO_SSO = _('The SSO feature is not enabled for this organization.')
+
+
+class OrganizationAuthProviderSendRemindersEndpoint(OrganizationEndpoint):
+    permission_classes = (OrganizationAdminPermission, )
+
+    def post(self, request, organization):
+        if not request.user.is_authenticated():
+            return Response(status=401)
```

Comment:
When would this case be hit? Isn't this already covered by permissions? You'd have no permission if you weren't logged in. Or is this to require the user to be logged in, instead of using an API key? If so, why?

Reply:
Thanks, this shouldn't be here.